projects/aws/bottlerocket-bootstrap/pkg/providers/snow/system/kernel.go

```diff
@@ -0,0 +1,24 @@
+package system
+
+import "github.com/pkg/errors"
+
+const (
+	maxSocketBufferSize = "8388608"
+	kernelCorePattern   = "/var/corefile/core.%e.%p.%h.%t"
+)
+
+func (s *Snow) configureKernelSettings() error {
+	if err := s.api.SetKernelRmenMax(maxSocketBufferSize); err != nil {
+		return errors.Wrap(err, "Error setting kernel maximum socket receive buffer size")
```

Comment:
this probably needs to change throughout tho
```suggestion
		return errors.Wrap(err, "setting kernel maximum socket receive buffer size")
```

Reply:
i will have a separate pr for this